js/templates/message-list-item.html

```diff
@@ -5,7 +5,7 @@
 	<div class="mail-message-header">
 		<div class="sender-image avatardiv">
 			{{#if senderImage}}
-			<img src="{{senderImage}}" width="32px" height="32px" />
+			<img src="{{senderImage}}" width="32px" height="32px" style="object-fit: cover;"/>
```

Comment:
@sagorika1996 I think you might want to adjust the SCSS rather than inlining this style in the HTML :wink: 

A selector like `.sender-image img` should work for this specific avatar image. The `object-fit` from `.avatar` could be removed then, I assume.

Reply:
@ChristophWurst Sender's pic in the autocomplete suggestions is also stretched https://github.com/nextcloud/mail/blob/f2ecf4d4ddd011475f991509d9f45af6d0c12607/js/views/composerview.js#L522-L524
That's why I included `object-fit` in `.avatar`